src/main/java/com/ericsson/ei/handlers/EventHandler.java

```diff
@@ -52,7 +52,7 @@ public RulesHandler getRulesHandler() {
     }
 
     public void eventReceived(String event) {
-        log.info("Thread id " + Thread.currentThread().getId() + " spawned");
+        LOGGER.info("Thread id {} spawned", Thread.currentThread().getId());
```

Comment:
Should this maybe be debug, the test loggs are full with those spawning logs.

Reply:
Sure, I'll make it debug.